### DISCUSSION TITLE
Software creation ideas from scratch beginning

### DISCUSSION THREAD

**Discussion by Msbeginner** [practical-tutorials/project-based-learning#435] [General]
Hello all
Looking for ideas for the creation of a software like time table creation.

**Comment by weshaan** [comment]
what exactly are you looking forward to making?

**Reply by Magicfingerz98** [reply]
Hello

**Reply by weshaan** [reply]
hey @Msbeginner

**Comment by aarushiksk** [comment]
Hi, I have a great idea, let's use a bot that can generate a time table according to the provided study material. We can use NodeJS or LangChain that can interact with OpenAI API to create this time table. However, there might be a paid API required depending on the number of tokens/words we want to generate.

**Comment by Tech-Fear-404** [comment]
I want to learn while building project. My main learnings include system
design and frontend development

**Comment by Tech-Fear-404** [comment]
Can you tell something free as I am a student and doesn't have that much
finanaces

**Comment by oxSTERBENxo** [comment]
❤️
